experimental/packages/otlp-grpc-exporter-base/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@opentelemetry/otlp-grpc-exporter-base",
+  "version": "0.27.0",
+  "description": "OpenTelemetry Exporter base for gRPC",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js",
+  "scripts": {
+    "prepublishOnly": "npm run compile",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+    "compile": "tsc --build",
+    "clean": "tsc --build --clean",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "postcompile": "npm run submodule && npm run protos:copy",
+    "protos:copy": "cpx protos/opentelemetry/**/*.* build/protos/opentelemetry",
+    "submodule": "git submodule sync --recursive && git submodule update --init --recursive",
+    "tdd": "npm run test -- --watch-extensions ts --watch",
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.test.ts'",
+    "version": "node ../../../scripts/version-update.js",
+    "watch": "npm run protos:copy && tsc -w",
+    "precompile": "lerna run version --scope $(npm pkg get name) --include-dependencies",
+    "prewatch": "npm run precompile"
+  },
+  "keywords": [
+    "opentelemetry",
+    "nodejs",
+    "grpc",
+    "tracing",
+    "profiling",
+    "metrics",
+    "stats"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=8.12.0"
+  },
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.js.map",
+    "build/src/**/*.d.ts",
+    "build/protos/**/*.proto",
+    "doc",
+    "LICENSE",
+    "README.md"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@babel/core": "7.16.0",
+    "@opentelemetry/api": "^1.0.0",
+    "@types/mocha": "8.2.3",
+    "@types/node": "14.17.33",
+    "@types/sinon": "10.0.6",
+    "codecov": "3.8.3",
+    "cpx": "1.5.0",
+    "mocha": "7.2.0",
+    "nyc": "15.1.0",
+    "rimraf": "3.0.2",
+    "sinon": "12.0.1",
+    "ts-loader": "8.3.0",
+    "ts-mocha": "8.0.0",
+    "typescript": "4.4.4",
+    "@opentelemetry/sdk-trace-base": "1.1.1",
+    "@opentelemetry/resources": "1.1.1",
+    "@opentelemetry/exporter-trace-otlp-http": "0.27.0"
```

Comment:
For now, yes, as it is still using the types from the trace exporter for tests. However, this is just a dev dependency. It can be removed in the next step, where we will use the `otlp-transfomer` package instead (as mentioned in #2886). :slightly_smiling_face: